test/parsing/files/delimiter.equal.valid.json

```diff
@@ -14,7 +14,7 @@
          "properties": {
             "--ignore": {
                "id": "--ignore",
-               "type": "string",
+               "type": "null",
```

Comment:
`null` is useg without quotations

Reply:
In our EP (https://github.com/repometric/ep/blob/master/ep/lh/ep-lh-0003.md) we have `"type": "null"`. Is it mistake in EP?

Reply:
Yes

Reply:
@svoboda-rabstvo i fixed it. Please, recheck PR.